R/projpred.R

```diff
@@ -13,8 +13,13 @@
 #' (see \code{\link[projpred:get_refmodel]{get_refmodel}} for details).
 #' If \code{NULL} (the default), \code{cvfun} is defined internally
 #' based on \code{\link{kfold.brmsfit}}.
+#' @param dis Passed to argument \code{dis} of
```

Comment:
In which cases would this have to be added manually?

Reply:
For the latent projection with the probit or the logit link, projpred can use an internal default (constantly 1 and 1.6, respectively), but for other links, there is no such internal default. Perhaps users can come up with some reasonable `dis` value(s) in the future, so it's better to keep this flexible. Apart from that, `dis` is an argument of `init_refmodel()` and previously, it wasn't possible to specify it when using `get_refmodel.brmsfit()`, so already from a conceptual point of view, I think it makes sense to keep this flexible.

Reply:
I see. Makes sense.